src/Dfe.PlanTech.Web/Views/Recommendations/Pagination.cshtml

```diff
@@ -0,0 +1,18 @@
+@using Dfe.PlanTech.Domain.Questionnaire.Interfaces;
+@model (IHeaderWithContent? Previous, IHeaderWithContent? Next)
```

Comment:
I didn't realise you could pass in both like this, really nice, thanks for this improvement